<?xml version="1.0"?>
<launch>
    <include file="$(find ropod_navigation_test)/launch/ropod_navigation_maneuvers.launch"/> 
   
    


    
    
    <node name="demo_navigation" pkg="ropod_demo_dec_2017" type="demo_navigation" output="screen">
        <param name="move_base_server" value="/move_base" />
        <param name="move_base_feedback_topic" value="/move_base/feedback" />
        <param name="move_base_cancel_topic" value="/move_base/cancel" />        
        
        <rosparam file="$(find ropod_navigation_test)/config/parameters/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find ropod_navigation_test)/config/parameters/costmap_common_params.yaml" command="load" ns="local_costmap" />    
        <rosparam file="$(find ropod_navigation_test)/config/parameters/footprint_ropod.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find ropod_navigation_test)/config/parameters/footprint_ropod.yaml" command="load" ns="local_costmap" />       
        <rosparam file="$(find ropod_navigation_test)/config/parameters/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find ropod_navigation_test)/config/parameters/local_costmap_params.yaml"  command="load"/>    
        <rosparam file="$(find ropod_navigation_test)/config/parameters/base_local_planner_params.yaml" command="load" />
                    
    </node>
        
    
</launch>
